lighthouse-cli/test/smokehouse/firehouse.js

```diff
@@ -14,8 +14,8 @@
 /* eslint-disable no-console */
 
 const log = require('lighthouse-logger');
-const smokeTests = require('./smoke-test-dfns.js');
-const {collateResults, report} = require('./smokehouse-report.js');
+const smokeTests = require('./test-definitions/core-tests.js');
+const report = require('./report-assert.js');
```

Comment:
(commenting here for no reason)

Firehouse should move into runners, right?  

(i wouldnt mind changing the name now because i totally forget what its there for)

Reply:
`--runner` needs a concrete way to run Lighthouse, but Firehouse doesn't even  provide that. You'd use it to create what would be a frontend, one capable of tweaking expectations.

AFAIU if a 'runner' could also tweak expectations like firehouse.js does, we can delete firehouse.js

Reply:
> AFAIU if a 'runner' could also tweak expectations like firehouse.js does, we can delete firehouse.js

I'll follow up with my proposal to get all these needs addressed. Should be a small (hopefully negative) diff :)

Reply:
gotcha that makes sense.   almost like a lib/expectation-filterer.js 

or wait... EXPECTATION-SIEVE.JS

hot

Reply:
`smoke-mask` 😷